FDB failed with error, it looks like hibernate-envers/5.4.24.Final is not present in nexus @akumar074 
```
[2026-03-18T16:01:39.362Z] [ERROR] Failed to execute goal on project kie-wb-common-data-modeller-core: Could not resolve dependencies for project org.kie.workbench.services:kie-wb-common-data-modeller-core:jar:7.67.2-SNAPSHOT: Failed to collect dependencies at org.hibernate:hibernate-envers:jar:5.4.24.Final: Failed to read artifact descriptor for org.hibernate:hibernate-envers:jar:5.4.24.Final: Could not transfer artifact org.hibernate:hibernate-envers:pom:5.4.24.Final from/to mirror-central (https://baqe-nexus.apps.int.prod-scale-spoke1-aws-us-east-1.itup.redhat.com/nexus/content/groups/kie-all/): transfer failed for https://baqe-nexus.apps.int.prod-scale-spoke1-aws-us-east-1.itup.redhat.com/nexus/content/groups/kie-all/org/hibernate/hibernate-envers/5.4.24.Final/hibernate-envers-5.4.24.Final.pom: Remote host terminated the handshake: SSL peer shut down incorrectly -> [Help 1]
[2026-03-18T16:01:39.362Z] org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal on project kie-wb-common-data-modeller-core: Could not resolve dependencies for project org.kie.workbench.services:kie-wb-common-data-modeller-core:jar:7.67.2-SNAPSHOT: Failed to collect dependencies at org.hibernate:hibernate-envers:jar:5.4.24.Final
```

--- a/pom.xml
+++ b/pom.xml
@@ -4684,11 +4684,8 @@
         <version>${version.org.hamcrest}</version>
       </dependency>
 
-      <dependency>
-        <groupId>org.hibernate</groupId>
-        <artifactId>hibernate-envers</artifactId>
-        <version>${version.org.hibernate}</version>
-      </dependency>
+      <!-- hibernate-envers/5.4.24.Final excluded - not available in BAMOE QE nexus -->
+      <!-- See exclusions in hibernate-core, hibernate-entitymanager, and hibernate-jpamodelgen -->
       <dependency>
         <groupId>org.hibernate.common</groupId>
         <artifactId>hibernate-commons-annotations</artifactId>
@@ -4702,6 +4699,10 @@
           <exclusion>
             <groupId>javax.xml.bind</groupId>
             <artifactId>jaxb-api</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-envers</artifactId>
           </exclusion>
         </exclusions>
       </dependency>
@@ -6215,6 +6216,10 @@
             <groupId>javax.xml.bind</groupId>
             <artifactId>jaxb-api</artifactId>
           </exclusion>
+          <exclusion>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-envers</artifactId>
+          </exclusion>
         </exclusions>
       </dependency>
       <dependency>
@@ -6229,6 +6234,10 @@
           <exclusion>
             <groupId>org.jboss.spec.javax.transaction</groupId>
             <artifactId>jboss-transaction-api_1.2_spec</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-envers</artifactId>
           </exclusion>
         </exclusions>
       </dependency>

